src/python/automation/build_named_fault_mfd_index.py

```diff
@@ -42,18 +42,21 @@ def build(self):
                 for filename in fnmatch.filter(files, pattern):
                     folder_path = PurePath(root)
                     if len(folder_path.parts) - len(PurePath(self._dir_name).parts) == 1:
-                        #print(root, filename)
+                        print(root, filename)
```

Comment:
are you checking in some debug code?

Reply:
Yeah,  well the whole thing is on constant debug!. It need some work to move configurations out of scripts and provide some test coverage but I haven't got the time ATM.